src/actions/channel.js

```diff
@@ -98,13 +102,22 @@ export const loadChannelMembers = () => (dispatch, getState) => {
   })
 
   api
-    .listMembers(channel.id)
-    .then(res => res.results)
-    .then(users => users.map(normalizeUserData))
-    .then(payload => {
+    .listMembers(channel.id, {
+      limit: 50,
```

Comment:
Ideally the limit here and in the component use the same imported constant since they are connected and if you would change one without the other it would lead to a bug.